lambda/qnabot-common-layer/qnabot/logging.js

```diff
@@ -93,7 +87,6 @@ async function _getPIIEntities(params){
         try{
            return JSON.parse(process.env.comprehendResult)
         }catch(e){
-            console.warn("invalid JSON in process.env.comprehend",e)
```

Comment:
What is left in this catch after then removing warning message? Any handling logging needed?

If we don't want the content the exception to on console.warn, should we keep the reset of the message?

Reply:
We cache the comprehend result in an environment variable so we don't have to call Comprehend again. If the content of the environment variable isn't JSON, it just calls Comprehend again.

That was before I had the isJSON function in the same layer. Using isJSON would be more idiomatic.